dom/exercise1.js

```diff
@@ -10,4 +10,23 @@
     - p tag
     - class name
     - id name
-*/
\ No newline at end of file
+*/
+
+const singleP = document.querySelector("p");
+console.log(singleP);
+
+const singleClass = document.querySelector("h1.headers");
```

Comment:
This means select all h1 with the class of headers. We want to get all classes regardless of what element that class is attached to

Reply:
is using ".headers" the correct syntax ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector

Reply:
I swear I read that on the docs but must have misunderstood something. my apologies, it has been corrected and a commit has been sent